frontend/packages/multicluster-sdk/src/api/useFleetK8sWatchResource.ts

```diff
@@ -135,12 +135,6 @@ export function useFleetK8sWatchResource<R extends FleetK8sResourceCommon | Flee
         return
       }
 
-      // Check if we already have this resource cached and it's not expired
-      const existingEntry = store.getResource(requestPath)
-      if (existingEntry && !store.isResourceExpired(requestPath)) {
-        return
-      }
```

Comment:
Exiting the code under these conditions and before checking for an active socket for the resource was a mistake. The new code will check to see if a fetch is necessary but will also continue on to [check for an open and ready socket for the resource](https://github.com/stolostron/console/pull/5189/files#diff-e8f92a33a6b70d54d07d6b90b42b71d24e733a67a47d19ccb4769d2566989d9bR141-R144).